Tiltfile

```diff
@@ -169,125 +214,203 @@ WORKDIR /
 COPY --from=tilt-helper /start.sh .
 COPY --from=tilt-helper /restart.sh .
 COPY --from=tilt-helper /go/bin/dlv .
-COPY manager .
+COPY $binary_name .
 """
+def build_go_binary(name, context, reload_deps, debug, go_main, binary_name, label):
```

Comment:
Looks like binary_name is not used.

(there are probably still to many places where "manager" is hard-coded instead of using the binary_name (?))